be/src/exec/vectorized/analytor.cpp

```diff
@@ -375,7 +375,7 @@ void Analytor::reset_window_state() {
     }
 }
 
-void Analytor::get_window_function_result(int32_t start, int32_t end) {
```

Comment:
Is it means maybe we have some large binary column problems or large int column.
```
    bool reach_capacity_limit() const override { return _data.size() > Column::MAX_CAPACITY_LIMIT; }

```

Reply:
```
void Analytor::get_window_function_result(size_t start, size_t end) {
    DCHECK_GT(end, start);
    for (size_t i = 0; i < _agg_fn_ctxs.size(); i++) {
        vectorized::Column* agg_column = _result_window_columns[i].get();
        _agg_functions[i]->get_values(_agg_fn_ctxs[i], _managed_fn_states[0]->data() + _agg_states_offsets[i],
                                      agg_column, start, end);
    }
}
```

the `start`, `end` of  function `get_values` is `size_t`

Reply:
so we keep consistent in the interface with get_values